src/components/study/Registration.vue

```diff
@@ -2,17 +2,21 @@
   <div>
     <MainNav title="Install The App" :back-to-overview="true" :show-help="true" :show-steps="true"/>
     <div class="container">
-      <p>Thank you! Please enter your mobile phone number <!--and select your phone type--> to receive your download link for the mPower app:</p>
+      <p>Thank you! Please enter your mobile phone number and select your phone type to receive your download link for the mPower app:</p>
 
-      <p style="text-align: center; margin-top: 2rem">
-        <mdc-textfield ref="phoneField" v-model="phone" label="Enter phone number" type="tel" pattern="[0-9]*"/>
-      </p>
-      <p v-if="showAndroid" style="text-align: center; margin-top: 3rem">Pick one depending on your mobile phone:</p>
+      <div style="text-align: center; margin-top: 2rem">
+        <vue-tel-input
```

Comment:
Use vue-tel-input, capture the user provided phone number in phoneValue, set the default country based on the locale, and restrict the available countries to the 2 we say are eligible in this study in this locale (US and Canada, or US and Netherlands)

Reply:
When the country is changed, call onCountryChange()